src/CreatePostWindow.tsx

```diff
@@ -0,0 +1,90 @@
+import "./CreatePostWindow.css";
+import React, { useState, Component } from "react";
+
+function CreatePostWindow(props: {
+  OnCreate: (_: string) => void;
```

Comment:
all the functions are in uppercase, it has to be lowercase letter. only name of components are in upper case

Reply:
and i would like to see better naming, like onCreate, create what? setTemp, which temp? and so on

Reply:
> all the functions are in uppercase, it has to be lowercase letter. only name of components are in upper case

For some reason I was lead to believe that every objects comes from **props** has to be uppercase - I will change it


~~


> and i would like to see better naming, like onCreate, create what? setTemp, which temp? and so on

I didn't want to deal with too long names... Won't it be clear by the context (role of the component and so on)?

Reply:
Not really, it could be something clear from the context, but it could be something else either...